Packs/PAN-OS/Playbooks/playbook-PAN-OS_-_Block_IP_-_Static_Address_Group_README.md

```diff
@@ -1,39 +1,50 @@
-Blocks IP addresses using Static Address Groups in Palo Alto Networks Panorama or Firewall.
+This playbook blocks IP addresses using Static Address Groups in Palo Alto Networks Panorama or Firewall.
 The playbook receives malicious IP addresses and an address group name as inputs, verifies that the addresses are not already a part of the address group, adds them and commits the configuration.
 
-**Note**: The playbook does not block the address group communication using a policy block rule. This step will be taken once outside of the playbook.
+***Note - The playbook does not block the address group communication using a policy block rule. This step will be taken once outside of the playbook.
 
 ## Dependencies
+
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
 ### Sub-playbooks
+
 * PAN-OS Commit Configuration
 
 ### Integrations
-This playbook does not use any integrations.
+
+* Panorama
 
 ### Scripts
+
 This playbook does not use any scripts.
 
 ### Commands
+
 * pan-os-create-address
 * pan-os-edit-address-group
 * pan-os-get-address-group
 
 ## Playbook Inputs
+
 ---
 
-| **Name** | **Description** | **Default Value** | **Source** | **Required** |
-| --- | --- | --- | --- | --- |
-| IP | The IP address to block. | Address | IP | Optional |
-| LogForwarding | The Panorama log forwarding object name. | - | - | Optional |
-| AddressGroupName | The static address group name. | Demisto Remediation - Static Address Group | - | Optional |
-| AutoCommit | Whether to commit the configuration automatically. "Yes" will commit automatically. "No" will commit manually. | No | - | Optional |
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| IP | IP address to block | IP.Address | Optional |
+| AddressGroupName | Static address group name | Demisto Remediation - Static Address Group | Optional |
+| AutoCommit | This input establishes whether to commit the configuration automatically.<br/>Yes - Commit automatically.<br/>No - Commit manually. | No | Optional |
+| IPDescription | The description of the IP in case it will get created by the playbook. | Automatically created using "PAN-OS - Block IP - Static Address Group" playbook in XSOAR. | Optional |
```

Comment:
```suggestion
| IPDescription | The description of the IP in case it will get created by the playbook. | Automatically created using "PAN-OS - Block IP - Static Address Group" playbook in Cortex XSOAR. | Optional |
```